config/environments/development.rb

```diff
@@ -15,8 +15,24 @@
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
 
-  # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  # Raise hell if the mailer can't send.
+  config.action_mailer.raise_delivery_errors = true
+
+  # Mailer settings
+  config.action_mailer.delivery_method = :smtp
+
+  config.action_mailer.smtp_settings = {
+  address: “smtp.gmail.com”,
+  port: 587,
+  domain: ENV[“GMAIL_DOMAIN”],
+  authentication: “plain”,
+  enable_starttls_auto: true,
+  user_name: ENV[“GMAIL_USERNAME”],
+  password: ENV[“GMAIL_PASSWORD”]
+  }
+
+  # Set the mailer host
+  config.action_mailer.default_url_options = { :host => 'localhost:3000' }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.